pkg/tests/dbaas_operator_tests.go

```diff
@@ -150,10 +162,9 @@ var _ = Describe("Rhoda e2e Test", func() {
 								}
 							}
 							return false
-						} else {
-							fmt.Println("inventory.Status.Conditions Len is 0")
-							return false
 						}
+						fmt.Println("inventory.Status.Conditions Len is 0")
+						return false
```

Comment:
Not sure why this change was made. I find the previous format much more readable. You know that the log statement is printed in the else, if Conditions Len is NOT > 0... explicit If/Else makes it easier to understand what goes on, versus an implicit else because the if ends with a return.

Reply:
This is actually a common practice for go and is inforced by golint: when the if statement ends with a "return", the "else" statement is redundant so for simplicity it is removed here. 

Anyway, I will update this PR to only include the changes for adding the BeforeEach(), and all format-related changes will be done by a separate PR.